### ISSUE TITLE
write_codemeta() errors on a minimal package

### ISSUE THREAD

**Issue by ThierryO** [ropensci/codemetar#298]
I've traced the error in to be in [`github_path`](https://github.com/ropensci/codemetar/blob/e5f2555a720b830c9a81d69a5a31e2c51f4fb6de/R/guess_github_metadata.R#L47).
In my example `cm$codeRepository` is `NULL` which triggers an error in `urltools::domain()`.

reprex
```
remotes::install_github("inbo/checklist")
checklist::create_package()
codemetar::write_codemeta()
```

**Comment by maelle** [comment]
Thank you, good catch! Would you be interested in making a PR fixing the problem? (to the [dev branch](https://github.com/ropensci/codemetar/tree/dev))

Also, could you share the link to your package repo, out of curiosity?

**Comment by ThierryO** [comment]
Here is the [repo](https://github.com/inbo/checklist). I've assumed you could derive it from the `remotes::install_github()`.

I'll try to propose a quick (and possible dirty) fix.

**Comment by maelle** [comment]
>  I've assumed you could derive it from the remotes::install_github().

Sorry :woman_facepalming:

**Comment by ThierryO** [comment]
Is it OK to add this fix to #300 or do you prefer a different PR?
